docs/reference/notificationsconfiguration.md

```diff
@@ -0,0 +1,92 @@
+# NotificationsConfiguration
+
+The `NotificationsConfiguration` resource is a Kubernetes Custom Resource (CRD) that allows users to add the triggers, templates, services and subscriptions to the Argo CD Notifications Configmap.
+
+A `NotificationsConfiguration` custom resource with name `default-notifications-configuration` is created **OOTB** with default configuration. Users should update this custom resource with their templates, triggers, services, subscriptios or any other configuration.
```

Comment:
Should we add an explicit important note here, stating that only updates to `default-notifications-configuration` are supported, and users cannot create new NotificationsConfiguration custom resources as of now?